packages/teams-js/src/public/dialog.ts

```diff
@@ -33,6 +33,23 @@ export namespace dialog {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   export type PostMessageChannel = (message: any) => void;
   export type DialogSubmitHandler = (result: ISdkResponse) => void;
+  const storedMessages: string[] = [];
+
+  /**
+   * @hidden
+   * Hide from docs because this function is only used during initialization
+   * ------------------
+   * Adds register handlers for messageForChild upon initialization and only in the tasks FrameContext. {@link FrameContexts.task}
+   * Function is called in {@link app.initializeHelper}
```

Comment:
Does this link actually go to app.initializeHelper? I thought, we would need to have 'app' imported in this file to make the link.

Reply:
Good call, I'm going to remove the link because these docs are hidden anyways